### PR TITLE
Tox tests

### PR DESCRIPTION
**Pull request by tswfi**
Add Tox test suite (prereq for fixing python3, py36 tests will for now ;)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,8 @@ scripts/privkey.pem
 scripts/run_callback.sh
 
 *.egg-info*
+
+.tox/
+*.pyc
+
+Pipfile.lock
```
Comment (ooz) [inline .gitignore]: Hm, this is actually in my global gitignore, so i missed it here. Nice find!

**Review by ooz — APPROVED** [review]

**Comment by ooz** [comment]
Hm, PR testing via travis-ci will never work here, because they don't expose private environment variables and tests demand API access to ePages developer shops. There's room for improvement here. :|

**Comment by tswfi** [comment]
Would probably need to mock the services to get the tests going through.. but thats quite a lot of work :)

**Comment by ooz** [comment]
True, and I am not keen on investing that amount of time into the project at this stage. ;)

A quick win would be separating unit tests from test that run against the ePages API.
Then run just unit tests for PRs and all tests for master branch/pypi releases.
But currently unit tests are only used for the signature validation.

Please note, that I am using ePages dev shops that are provisioned via the publicly available sign-up on https://developer.epages.com/ to run the tests.

**Pull request merged**